modules/ListView.js

```diff
@@ -366,31 +405,74 @@
       return model ? this.getChildView(this.__modelToViewMap[model[this.__modelId]]) : undefined;
     },
 
+    /**
+     * @returns {Boolean} returns true if there exists any generated item views
+     * @method hasItemViews
+     */
+    hasItemViews: function() {
+      return !_.isEmpty(this.getItemViews());
+    },
+
+    /**
+     * @returns {Array of views} Returns unordered list of views generated by this list view
+     * @method getItemViews
+     */
+    getItemViews: function() {
+      var view = this;
+      var orderedViewIds = _.map(this.__orderedModelIdList, function(modelId) { return view.__modelToViewMap[modelId]; });
```

Comment:
too much going on in this line :).

is there a method already that takes a modelId and returns the view id?  Just pass that into _.map() and I think map also has a convenience for the context.

    _.map(this.__orderedModelIdList, this.getViewIdFromModelId, this);

or something like that.